apps/services/license-api/src/app/modules/license/license.controller.ts

```diff
@@ -108,6 +114,8 @@ export class UserLicensesController {
 }
 
 @Controller({ version: ['1'], path: 'licenses/' })
+@UseGuards(IdsAuthGuard, LicenseTypeScopesGuard)
```

Comment:
Should use standard `ScopesGuard`?

Reply:
No I mean this controller should just use the standard ScopesGuard and @Scopes decorator since it doesn't need any dynamic License Type scope magic. This also simplifies the LicenseTypeScopeGuard further.

Reply:
ah ok, i understand